packages/base/all/vendor-config-onl/src/python/onl/install/ConfUtils.py

```diff
@@ -315,7 +316,15 @@ def install(self, device):
         elif self.bootPart:
             cmds.append("mpt=$(mktemp -t -d)")
             cmds.append("mount %s $mpt" % self.bootPart)
-            cmds.append(("sts=0; %s --boot-directory=\"$mpt\" %s || sts=$?"
+            if isUEFI:
+                cmds.append("[ -n \"$(efibootmgr -v | grep 'Open Network Linux')\" ] && (efibootmgr -b $(efibootmgr | grep \"Open Network Linux\" | sed 's/^.*Boot//g'| sed 's/** Open.*$//g') -B)")
```

Comment:
The system name can be configured via the sysconfig interface to something other than 'Open Network Linux'. 

We will integrate the initial UEFI support as-is and then revisit this for customization support.